### DISCUSSION TITLE
negative case handling!

### DISCUSSION THREAD

**Discussion by brundabharadwaj** [codinasion-archive/.github-re#17] [Q&A]
Hi @harshraj8843 ,
As part of review can we expect contributors to add negative case handling?
Is there any specific reason for not having those checks ?

- Ps pardon, query from a newbie 🙌🏻

**Comment by harshraj8843** [comment]
Yes, we can.

But not having them is not a problem as more contributors will be able to contribute to add new features to the solutions ( I think 🤔 )

**Reply by brundabharadwaj** [reply]
Sounds good :)
